website/docs/Pipelines/snM3C/README.md

```diff
@@ -6,87 +6,182 @@ slug: /Pipelines/snM3C/README
 
-The main input files and parameters include:
+#### Input descriptions
 
 | Parameter | Description |
 | ---| --- |
-| fastq_input_read1 | Array of multiplexed FASTQ files for read 1 |
-| fastq_input_read2 | Array of multiplexed FASTQ files for read 2 |
-| random_primer_indexes | File containing random primer indexes |
-| plate_id | String specifying the plate ID |
-| output_basename | String specifying a basename to be used for naming files |
-| tarred_index_files | File containing tarred index files for hisat-3 mapping |
-| mapping_yaml | File containing YAML configuration for mapping steps with snakemake | 
-| snakefile | File containing the snakefile for mapping |
-| chromosome_sizes | File containing chromosome sizes information |
-| genome_fa | File containing the reference genome in FASTA format | 
-
-
-## Tasks and Tools
-The workflow contains two tasks described below. The parameters and more details about these tools can be found in the [YAP documentation](https://hq-1.gitbook.io/mc/).
+| fastq_input_read1 | Array of multiplexed FASTQ files for read 1. |
+| fastq_input_read2 | Array of multiplexed FASTQ files for read 2. |
+| random_primer_indexes | File containing random primer indexes. |
+| plate_id | String specifying the plate ID. |
+| tarred_index_files | File containing tarred index files for hisat-3 mapping. |
+| genome_fa | File containing the reference genome in FASTA format. | 
+| chromosome_sizes | File containing the genome chromosome sizes. |
+| r1_adapter | Optional string describing the adapter sequence for read 1 paired-end reads to be used during adapter trimming with Cutadapt; default is "AGATCGGAAGAGCACACGTCTGAAC". |
+| r2_adapter | Optional string describing the adapter sequence for read 2 paired-end reads to be used during adapter trimming with Cutadapt; default is  "AGATCGGAAGAGCGTCGTGTAGGGA". |
+| r1_left_cut | Optional integer describing the number of bases to be trimmed from the beginning of read 1 with Cutadapt; default is 10. |
+| r1_right_cut | Optional integer describing the number of bases to be trimmed from the end of read 1 with Cutadapt; default is 10. |
+| r2_left_cut | Optional integer describing the number of bases to be trimmed from the beginning of read 2 with Cutadapt; default is 10. |
+| r2_right_cut | Optional integer describing the number of bases to be trimmed from the end of read 2 with Cutadapt; default is 10. |
+| min_read_length | Optional integer; if a read length is smaller than `min_read_length`, both paired-end reads will be discarded; default is 30.  |
+| num_upstr_bases | Optional integer describing the number of bases upstream of the C base to include in ALLC file context column created using ALLCools; default is 0. |
+| num_downstr_bases | Optional integer describing the number of bases upstream of the C base to include in ALLC file context column created using ALLCools; default is 2. |
```

Comment:
Not super familiar with this input, but should it be "number of bases downstream of the C base" ?

Reply:
Yes good catch! Thank you!